src/Makefile.am

```diff
@@ -102,13 +102,14 @@
 #                                                                         #
 #*************************************************************************#
 
-bin_PROGRAMS = drakvuf xen_memclone injector
+bin_PROGRAMS = drakvuf xen_memclone injector mem_monitor
```

Comment:
So this looks to me like it would be fine to have as a plugin instead of a standalone application.